### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11<br /><br />Workflow run https://github.com/creative-commoners/silverstripe-gridfieldextensions/actions?query=branch%3Apulls%2F3.3%2Fmodule-standards

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Please update badge in readme

**Comment by emteknetnz** [comment]
Updated

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SilverStripe Grid Field Extensions Module
 
-[![Build Status](https://travis-ci.org/symbiote/silverstripe-gridfieldextensions.svg?branch=master)](https://travis-ci.org/symbiote/silverstripe-gridfieldextensions)
+[![CI](https://github.com/silverstripe/silverstripe-gridfieldextensions/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/silverstripe-gridfieldextensions/actions/workflows/ci.yml)
```
Comment (ChloeSartorelli) [inline README.md]: symbiote, not silverstripe.
I didn't notice this on the other symbiote ones which i merged, but those are probably wrong too - will need new PRs to fix them (sorry)

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (emteknetnz) [reply README.md]: updated

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**